other/upgrade_2-1_mysql.sql

```diff
@@ -2632,8 +2632,8 @@ if ($doChange)
 
 ---# Add the new ip columns to members
 ALTER TABLE {$db_prefix}members
-ADD COLUMN member_ip VARBINARY(16),
-ADD COLUMN member_ip2 VARBINARY(16);
+ADD COLUMN member_ip VARBINARY(16) DEFAULT '',
```

Comment:
the default '' can't be used here,
since it's not a valid ip adress so we got issues in the long run (can't safe this kind of value in pg for exp.)
same for the other places

Reply:
maybe instead write here some ip adress into which is known to be place holder like: 127.255.255.255

Reply:
Got rid of the default.  Set it with an UPDATE statement instead.  Slower, but a bit safer.